src/components/Profile/Profile.styles.js

```diff
@@ -1,4 +1,4 @@
-import { makeStyles } from '@material-ui/core';
+import { makeStyles, Hidden } from '@material-ui/core';
```

Comment:
```suggestion
import { makeStyles } from '@material-ui/core';
```
Are you making use of `Hidden`? If not then please remove it.

Reply:
Removed ✌